src/components/Sticky/Sticky.tsx

```diff
@@ -53,8 +53,9 @@ export default class Sticky extends React.Component<Props, State> {
     const {style, isSticky} = this.state;
     const {children} = this.props;
 
-    const childrenContent =
-      typeof children === 'function' ? children(isSticky) : children;
```

Comment:
Any idea why this needed to change? Normally the type check is enough of a guard.

Reply:
I believe this is due to this [breaking change](https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#narrowing-functions-now-intersects--object-and-unconstrained-generic-type-parameters) in how narrowing works for when children can be either a function a ReactNode. We now have to explicitly narrow it ourself with as type guard